fix travis tests

The travis tests are all failing at the moment.  appveyor seems okay.

also, a subset of the tests fail:
```
============================= test session starts ==============================
platform darwin -- Python 3.6.6, pytest-3.6.4, py-1.6.0, pluggy-0.7.1
rootdir: /Users/travis/build/AllenInstitute/AllenSDK, inifile: setup.cfg
plugins: xdist-1.23.2, pep8-1.0.6, ordering-0.5, mock-1.10.0, forked-0.2, cov-2.6.0
collecting 62 items                                                            /Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_cacheable.py:55: FutureWarning: from_csv is deprecated. Please use read_csv(...) instead. Note that some of the default arguments are different, so please refer to the documentation for from_csv when changing your function calls
  """))
collecting 222 items                                                           /Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_pager.py:63: FutureWarning: from_csv is deprecated. Please use read_csv(...) instead. Note that some of the default arguments are different, so please refer to the documentation for from_csv when changing your function calls
  """))
collected 1002 items                                                           
allensdk/test/test_deprecated.py ..                                            
allensdk/test/test_inline_examples.py sssss                                    
allensdk/test/test_temp_dir.py ..                                              
allensdk/test/api/test_annotated_section_data_set_api.py ...                   
allensdk/test/api/test_api.py f......                                          
allensdk/test/api/test_biophysical_api.py ......                               
allensdk/test/api/test_brain_observatory_api.py ...............................
allensdk/test/api/test_cache.py ......                                         
allensdk/test/api/test_cacheable.py ..........                                 
allensdk/test/api/test_cell_types_api.py f..                                   
allensdk/test/api/test_file_download.py ........                               
allensdk/test/api/test_glif_api.py fffff                                       
allensdk/test/api/test_grid_data_api.py .....                                  
allensdk/test/api/test_image_download_api.py ......................................
allensdk/test/api/test_mouse_atlas_api.py .....                                
allensdk/test/api/test_mouse_connectivity_api.py ......................................................................
allensdk/test/api/test_ontologies_api.py ................                      
allensdk/test/api/test_pager.py .......                                        
allensdk/test/api/test_reference_space_api.py ......................................................
allensdk/test/api/test_rma_template.py ..........                              
allensdk/test/api/test_svg_api.py ...                                          
allensdk/test/api/test_synchronization_api.py ......                           
allensdk/test/api/test_tree_search_api.py ..                                   
allensdk/test/brain_observatory/test_circle_plots.py ...........               
allensdk/test/brain_observatory/test_dff.py .....                              
allensdk/test/brain_observatory/test_drifting_gratings.py .....                
allensdk/test/brain_observatory/test_locally_sparse_noise.py ..................
allensdk/test/brain_observatory/test_natural_movie.py ..                       
allensdk/test/brain_observatory/test_natural_scenes.py .....                   
allensdk/test/brain_observatory/test_notebook.py sssss                         
allensdk/test/brain_observatory/test_observatory_plots.py ssssssssssssssssssssss
allensdk/test/brain_observatory/test_roi_masks.py ...                          
allensdk/test/brain_observatory/test_session_analysis.py ssssss                
allensdk/test/brain_observatory/test_session_analysis_regression.py sssssssssss
allensdk/test/brain_observatory/test_static_gratings.py ..........             
allensdk/test/brain_observatory/test_stimulus_analysis.py .....                
allensdk/test/brain_observatory/test_stimulus_info.py ..s.............sss.......
allensdk/test/brain_observatory/receptive_field_analysis/test_chisquarerf.py ......................
allensdk/test/brain_observatory/receptive_field_analysis/test_fitgaussian2D.py ..............................
allensdk/test/config/test_config_single_file_json.py ...                       
allensdk/test/config/test_json_comments.py ..........                          
allensdk/test/config/test_manifest.py ....                                     
allensdk/test/config/test_multi_file_config.py ....                            
allensdk/test/config/test_pyconfig_parser.py ....                              
allensdk/test/core/test_brain_observatory_cache.py ......s..ssss               
allensdk/test/core/test_brain_observatory_nwb_data_set.py sssssssssssss........
allensdk/test/core/test_cell_filters.py sfff.....                              
allensdk/test/core/test_cell_types_cache_unit.py ....................................................................................................
allensdk/test/core/test_h5_utilities.py .....                                  
allensdk/test/core/test_json_utilities.py .....                                
allensdk/test/core/test_mouse_connectivity_cache.py .........f..............   
allensdk/test/core/test_mouse_connectivity_notebook.py s                       
allensdk/test/core/test_nwb_data_set.py sss..xs                                
allensdk/test/core/test_obj_utilities.py ..                                    
allensdk/test/core/test_reference_space.py .............                       
allensdk/test/core/test_reference_space_cache.py .............                 
allensdk/test/core/test_reference_space_notebook.py s                          
allensdk/test/core/test_simple_tree.py ....................                    
allensdk/test/core/test_sitk_utilities.py ........................................................................................................................................................................................
allensdk/test/core/test_structure_tree.py ..............................       
allensdk/test/ephys/test_extractor.py .........                                
allensdk/test/ephys/test_features.py ......................s                   
allensdk/test/model/test_biophysical_perisomatic.py s                          
allensdk/test/model/test_glif.py ffssss
=================================== FAILURES ===================================
_____________________________ test_failed_download _____________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_api.py:80: running the test CRASHED with signal 11
___________________________ test_list_cells_unmocked ___________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_cell_types_api.py:95: running the test CRASHED with signal 11
------------------------------- captured stdout --------------------------------
10-08 19:45 allensdk.api.api INFO     Downloading URL: http://api.brain-map.org/api/v2/data/query.json?q=model::Specimen,rma::criteria,[is_cell_specimen$eq'true'],products[name$in'Mouse Cell Types','Human Cell Types'],ephys_result[failed$eqfalse],rma::include,structure,cortex_layer,donor(transgenic_lines,organism,conditions),specimen_tags,cell_soma_locations,ephys_features,data_sets,neuron_reconstructions,cell_reporter,rma::options[num_rows$eq'all']
Downloading URL: http://api.brain-map.org/api/v2/data/query.json?q=model::Specimen,rma::criteria,[is_cell_specimen$eq'true'],products[name$in'Mouse Cell Types','Human Cell Types'],ephys_result[failed$eqfalse],rma::include,structure,cortex_layer,donor(transgenic_lines,organism,conditions),specimen_tags,cell_soma_locations,ephys_features,data_sets,neuron_reconstructions,cell_reporter,rma::options[num_rows$eq'all']
______________________ test_get_neuronal_model_templates _______________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_glif_api.py:62: running the test CRASHED with signal 11
___________________________ test_get_neuronal_models ___________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_glif_api.py:87: running the test CRASHED with signal 11
_______________________ test_get_neuronal_models_no_ids ________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_glif_api.py:95: running the test CRASHED with signal 11
___________________________ test_get_neuron_configs ____________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_glif_api.py:101: running the test CRASHED with signal 11
_______________________________ test_deprecated ________________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/api/test_glif_api.py:112: running the test CRASHED with signal 11
________________________ test_dataframe_query_unmocked _________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/core/test_cell_filters.py:253: running the test CRASHED with signal 11
____________________ test_dataframe_query_between_unmocked _____________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/core/test_cell_filters.py:268: running the test CRASHED with signal 11
_______________________ test_dataframe_query_is_unmocked _______________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/core/test_cell_filters.py:283: running the test CRASHED with signal 11
___________________________ test_filter_experiments ____________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/core/test_mouse_connectivity_cache.py:281: running the test CRASHED with signal 11
_____________________________ test_cache_stimulus ______________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/model/test_glif.py:127: running the test CRASHED with signal 11
_____________________________ test_run_glifneuron ______________________________
/Users/travis/build/AllenInstitute/AllenSDK/allensdk/test/model/test_glif.py:133: running the test CRASHED with signal 11
 generated xml file: /Users/travis/build/AllenInstitute/AllenSDK/test-reports/test.xml 
---------- coverage: platform darwin, python 3.6.6-final-0 -----------
Coverage HTML written to dir htmlcov
======== 13 failed, 904 passed, 84 skipped, 1 xfailed in 587.01 seconds ========
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -55,5 +55,5 @@ install:
   - pip install .
 
 script:
-  - make test
+  - py.test --cov=allensdk
   - codecov

